src/agent/debugapi.ts

```diff
@@ -59,10 +60,13 @@ export function create(
   if (singleton && !config_.forceNewAgent_) {
     return singleton;
   }
+  let debugapi: any;
```

Comment:
As we discussed offline - tests use a `forceNewAgent_` flag to overwrite the singleton with a new value. If the singleton is overwritten, you'll need to disconnect the old one first.